src/test/java/com/epam/tc/hw2/GeneralData.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.tc.hw2;
+
+public class GeneralData {
+    protected  String indexUrl = "https://jdi-testing.github.io/jdi-light/index.html";
+    protected  String expectedBrowserTitle = "Home Page";
+    protected String userName = "Roman";
+    protected String userPass = "Jdi1234";
+    protected String expectedUserLogin = "ROMAN IOVLEV";
```

Comment:
And how do declare string constant properly?

Reply:
Maybe private final